docs/overview/sapi.rst

```diff
@@ -92,7 +92,7 @@ runs these same configuration steps):
    file if needed.
 
 Alternatively, you can create and edit a
-:doc:`D-Wave Cloud Client configuration file </docs_cloud/sdk_index>`
+:ref:`D-Wave Cloud Client configuration file <configurationFiles>`
 manually.
```

Comment:
Typically we prefer inter-repo links to target higher-level content because content changes over time and you get broken links. In this case, the risk is low because (1) the source is the SDK so the doc build will break if the target changes (2) the link target name is unlikely to be duplicated elsewhere and (3) that section should have a long life. 
But you'll see references such as :doc:`D-Wave Cloud Client configuration file </docs_cloud/sdk_index>` in a lot of docstrings and I'd be cautious about changing such references in other repos.

Reply:
Thanks for the explanation.  For now I'd prefer to follow the usual practices.  Even though there are some safeties that you mention, it's only a minor clarity issue, so I went ahead and dropped this commit.